tests/h/services/list_organizations_test.py

```diff
@@ -9,19 +9,15 @@
 class TestListOrganizations:
     ALT_AUTHORITY = "bar.com"
 
-    def test_returns_all_organizations_if_no_authority_specified(
-        self, svc, organizations, alt_organizations
-    ):
+    def test_returns_all_organizations_if_no_authority_specified(self, svc):
         results = svc.organizations()
 
         names = [org.name for org in results]
 
         # The "Hypothesis" here is from the default org added by DB init
         assert names == ["Hypothesis", "alt_org_1", "alt_org_2", "org_1", "org_2"]
 
-    def test_returns_organizations_for_the_authority_specified(
-        self, svc, alt_organizations
-    ):
+    def test_returns_organizations_for_the_authority_specified(self, svc):
```

Comment:
:+1: Even if these are for "noise" they are already auto-used so this was double pointless.